### PR TITLE
:bug: Remove placeholders from descriptions

### PR DESCRIPTION
**Pull request by fabianvf**

### CODE REVIEW

File: default/generated/eap7/102-websphere-mqe.windup.yaml [hunk 1]
```diff
@@ -60,7 +60,7 @@
       pattern: com.ibm.mqe.adapters.MQe{type}Adapter
 - category: mandatory
   customVariables: []
-  description: IBM MQe com.ibm.mqe.administration.MQe{type}AdminMsg
+  description: The IBM MQe administration API, used to administer and monitor a WebSphere MQ Everyplace queue manager, provides various classes. In JBoss EAP 7, the JMS destinations can be configured with the JBoss Admin CLI.
```
Comment (jmle) [inline default/generated/eap7/102-websphere-mqe.windup.yaml:63]:
```suggestion
  description: The JMS destinations in IBM MQe administration API can be configured with the JBoss Admin CLI in JBoss EAP 7
```

File: default/generated/eap7/75-hibernate4-xml.windup.yaml [hunk 2]
```diff
@@ -40,8 +40,7 @@
       pattern: hibernate.transaction.factory_class
 - category: mandatory
   customVariables: []
-  description: Class org.hibernate.id.{idclass} was removed/deprecated in Hibernate
-    5
+  description: Classes from the 'org.hibernate.id' package were removed or deprecated in Hibernate 5. Use the 'org.hibernate.id.enhanced.SequenceStyleGenerator' instead.
```
Comment (jmle) [inline default/generated/eap7/75-hibernate4-xml.windup.yaml:43]:
```suggestion
  description: Classes from the 'org.hibernate.id' package were removed or deprecated in Hibernate 5
```

File: default/generated/eap7/76-hibernate4.windup.yaml [hunk 3]
```diff
@@ -672,7 +672,7 @@
   - name: idclass
     nameOfCaptureGroup: idclass
     pattern: org.hibernate.id.(?P<idclass>(TableGenerator|TableHiLoGenerator|SequenceGenerator|SequenceIdentityGenerator|SequenceHiLoGenerator))
-  description: Hibernate 5 - Removed deprecated class org.hibernate.id.{idclass}
+  description: Deprecated classes in Hibernate 5 under 'org.hibernate.id' are removed.  Use 'org.hibernate.id.enhanced.SequenceStyleGenerator' instead.
```
Comment (jmle) [inline default/generated/eap7/76-hibernate4.windup.yaml:675]:
```suggestion
  description: Deprecated classes in Hibernate 5 under 'org.hibernate.id' are removed
```

File: default/generated/eap7/82-hibernate51-53.windup.yaml [hunk 4]
```diff
@@ -466,7 +466,7 @@
   - name: packages
     nameOfCaptureGroup: packages
     pattern: org.hibernate.(?P<packages>(persister|tuple).)?.*
-  description: Hibernate 5.3 - SPI in package org.hibernate.{packages} changed
+  description: Changes occurred in SPI packages under 'org.hibernate'. Custom classes using these SPIs need to be reviewed and updated. Please refer to the Javadoc for these packages for details.
```
Comment (jmle) [inline default/generated/eap7/82-hibernate51-53.windup.yaml:469]:
```suggestion
  description: SPI packages under 'org.hibernate' have changed
```

File: default/generated/eap7/87-maven-javax-to-jakarta.rhamt.yaml [hunk 5]
```diff
@@ -72,7 +72,7 @@
       pattern: artifactId>activation<
 - category: potential
   customVariables: []
-  description: javax.{renamed-g} groupId has been replaced by jakarta.{renamed-g}
+  description: The groupId 'javax' has been replaced by 'jakarta' in JBoss EAP 7.3, or later. To ensure its Maven building, running or testing works as expected, use the corresponding Jakarta EE dependency.
```
Comment (jmle) [inline default/generated/eap7/87-maven-javax-to-jakarta.rhamt.yaml:75]:
```suggestion
  description: The groupId 'javax' has been replaced by 'jakarta' in JBoss EAP 7.3, or later
```

File: default/generated/eap7/94-weblogic-jms.windup.yaml [hunk 6]
```diff
@@ -122,7 +122,7 @@
   - name: type
     nameOfCaptureGroup: type
     pattern: oracle.jms.AQjms(?P<type>(Text|Stream|Object|Map|Bytes)?)?Message
-  description: Oracle JMS {type}Message
+  description: Oracle JMS messages Oracle JMS messages represent the actual data passed through JMS destinations. These references should be replaced with the Java EE standard API.
```
Comment (jmle) [inline default/generated/eap7/94-weblogic-jms.windup.yaml:125]:
```suggestion
  description: Oracle JMS messages references should be replaced with the Java EE standard API
```

File: default/generated/eap8/110-hibernate-6.2.windup.yaml [hunk 7]
```diff
@@ -74,7 +74,7 @@
...
-  description: Class {class-name} within the org.hibernate.cfg package has been moved
+  description: Classes within the 'org.hibernate.cfg' package have been moved. For more information, see the provided link.
```
Comment (jmle) [inline default/generated/eap8/110-hibernate-6.2.windup.yaml:77]:
```suggestion
  description: Classes within the 'org.hibernate.cfg' package have been moved
```

File: default/generated/eap8/110-hibernate-6.2.windup.yaml [hunk 8]
```diff
@@ -97,8 +97,7 @@
...
-  description: Class {class-name} within the org.hibernate.loader package has been
-    moved
+  description: Classes within the 'org.hibernate.loader' package have been moved. See link for more details.
```
Comment (jmle) [inline default/generated/eap8/110-hibernate-6.2.windup.yaml:100]:
```suggestion
  description: Classes within the 'org.hibernate.loader' package have been moved
```

File: default/generated/eap8/122-javax-to-jakarta-dependencies.windup.yaml [hunk 9]
```diff
@@ -20,8 +20,7 @@
       pattern: groupId>javax.(annotation|batch|ejb|el|enterprise.concurrent|enterprise.deploy|faces|interceptor|jms|jws|mail|management.j2ee|resource|security.auth.message|security.jacc|servlet|servlet.jsp|servlet.jsp.jstl|transaction|websocket|ws.rs|xml.bind|xml.registry|xml.rpc|xml.soap|xml.ws)<
 - category: mandatory
   customVariables: []
-  description: javax.{renamed-a} javax.{renamed-a}-api artifactId has been replaced
-    by jakarta.{renamed-a} jakarta.{renamed-a}-api
+  description: The artifactIds starting with javax.*-api are updated to their corresponding jakarta.*-api versions in dependency files.
```
Comment (jmle) [inline default/generated/eap8/122-javax-to-jakarta-dependencies.windup.yaml:23]:
```suggestion
  description: The artifactIds starting with javax.*-api must be updated to their corresponding jakarta.*-api versions in dependency files
```

File: default/generated/fuse-service-works/141-soa-p-5.windup.yaml [hunk 10]
```diff
@@ -854,7 +854,7 @@
       namespaces: {}
       xpath: //*[local-name()='action' and @class='org.jboss.soa.esb.actions.scripting.ScriptingAction']/@class
 - customVariables: []
-  description: ESB {javaclassname}
+  description: JBoss ESB uses certain actions to aggregate a message sequence into a single aggregated message. Fuse Service Works utilizes Camel routing and Camel's aggregator for this purpose. For more information, see the Camel Aggregator Documentation.
```
Comment (jmle) [inline default/generated/fuse-service-works/141-soa-p-5.windup.yaml:857]:
```suggestion
  description: JBoss ESB message aggregation must be changed to Camel routing
```

File: default/generated/openjdk11/145-java-removals.windup.yaml [hunk 11]
```diff
@@ -91,8 +91,7 @@
   - name: methodName
     nameOfCaptureGroup: methodName
     pattern: java.lang.SecurityManager.(?P<methodName>(checkAwtEventQueueAccess|checkSystemClipboardAccess|checkTopLevelWindow))?(.*)
-  description: AWT SecurityManager method java.lang.SecurityManager.{method-name}
-    has been removed in Java 11
+  description: Certain methods under `java.lang.SecurityManager` tied to the AWT stack have been removed in Java 11 due to modularisation. Methods such as `checkAwtEventQueueAccess()`, `checkSystemClipboardAccess()` and `checkTopLevelWindow(java.lang.Object)` need to be replaced with `java.lang.SecurityManager.checkPermission(java.security.Permission)`. Specific replacement methods are detailed in the 'message' field.
```
Comment (jmle) [inline default/generated/openjdk11/145-java-removals.windup.yaml:94]:
```suggestion
  description: Certain methods under `java.lang.SecurityManager` tied to the AWT stack have been removed in Java 11
```

File: default/generated/openjdk17/149-removed-classes.windup.yaml [hunk 12]
```diff
@@ -3,7 +3,7 @@
   - name: className
     nameOfCaptureGroup: className
     pattern: java.util.jar.(?P<className>(Pack200|Pack200.Packer|Pack200.Unpacker))
-  description: The java.util.jar.{class-name} class has been removed
+  description: The classes under java.util.jar have been removed after OpenJDK 11 and completely removed in OpenJDK 14. Please consider switching to either the jlink tool or the jpackage tool to create optimized, application-specific runtimes.
```
Comment (jmle) [inline default/generated/openjdk17/149-removed-classes.windup.yaml:6]:
```suggestion
  description: The classes under java.util.jar have been removed
```

File: default/generated/openjdk17/151-security-manager-deprecation.windup.yaml [hunk 13]
```diff
@@ -3,7 +3,7 @@
   - name: className
     nameOfCaptureGroup: className
     pattern: java.security.(?P<className>(AccessController|AccessControlContext|AccessControlException|DomainCombiner|Policy|PolicySpi|Policy.Parameters))
-  description: The java.security.{class-name} class has been deprecated
+  description: The Java Security Manager class has been deprecated The Java Security Manager API is deprecated in Java 17. Any references to it should be removed. See the link for further information.
```
Comment (jmle) [inline default/generated/openjdk17/151-security-manager-deprecation.windup.yaml:6]:
```suggestion
  description: The Java Security Manager class has been deprecated.
```

File: default/generated/openjdk17/151-security-manager-deprecation.windup.yaml [hunk 14]
```diff
@@ -22,7 +22,7 @@
   - name: fqcn
     nameOfCaptureGroup: fqcn
     pattern: (?P<fqcn>(java.lang.SecurityManager|java.rmi.RMISecurityManager|javax.security.auth.SubjectDomainCombiner))
-  description: The {fqcn} class has been deprecated
+  description: The class in question has been deprecated This references to the Java Security Manager API classes that were deprecated in Java 17. Any references should be removed. Further information can be found through the provided link.
```
Comment (jmle) [inline default/generated/openjdk17/151-security-manager-deprecation.windup.yaml:25]:
```suggestion
  description: References to the Java Security Manager API have been deprecated
```

File: default/generated/openjdk21/08-utf-8-by-default.windup.yaml [hunk 15]
```diff
@@ -3,7 +3,7 @@
   - name: className
     nameOfCaptureGroup: className
     pattern: java.io.(?P<className>(FileReader|FileWriter|InputStreamReader|OutputStreamWriter|PrintStream))?.*
-  description: The java.io.{class-name} constructor uses UTF-8 by default
+  description: The 'java.io' constructor defaults to the UTF-8 character If UTF-8 is not appropriate for your class, make sure to specify the appropriate character set.
```
Comment (jmle) [inline default/generated/openjdk21/08-utf-8-by-default.windup.yaml:6]:
```suggestion
  description: The 'java.io' constructor defaults to UTF-8
```

File: default/generated/openjdk21/08-utf-8-by-default.windup.yaml [hunk 16]
```diff
@@ -25,7 +25,7 @@
   - name: classNames
     nameOfCaptureGroup: classNames
     pattern: java.util.(?P<classNames>(Formatter|Scanner))?.*
-  description: The java.util.{class-names} constructor uses UTF-8 by default
+  description: The `java.util` constructor defaults to UTF-8 If UTF-8 is not suitable, specify the appropriate character set during the constructor call.
```
Comment (jmle) [inline default/generated/openjdk21/08-utf-8-by-default.windup.yaml:28]:
```suggestion
  description: The `java.util` constructor defaults to UTF-8
```

**Comment by rromannissen** [comment]
> I have done some modifications that I think improve a bit the descriptions, mostly shortening them (suggestions and references to links are usually placed in the message)

+1 to all @jmle suggestions!

File: default/generated/eap6/33-jboss-eap5-java.windup.yaml [hunk 17]
```diff
@@ -83,7 +83,7 @@
   - name: package
     nameOfCaptureGroup: package
     pattern: org.jboss(?P<package>(\.ejb3)?\.annotation(\.ejb)?.)?Management
-  description: org.jboss{package}.Management
+  description: The `@Management` annotation in 'org.jboss' package, is a JBoss EJB 3.0 extension, that wraps the bean as an MBean and installs it in the JBoss MBean Server. For singleton implementation, use the EJB3.1 spec `@Singleton` bean instead.
```
Comment (pranavgaikwad) [inline default/generated/eap6/33-jboss-eap5-java.windup.yaml:86]:
```suggestion
  description: Replace JBoss EJB 3.0 `@Management` annotation with EJB3.1 spec `@Singleton` bean
```

**Review by pranavgaikwad — APPROVED** [review]

**Pull request merged**